homeassistant/components/supla/__init__.py

```diff
@@ -22,11 +28,15 @@
     "LIGHTSWITCH": "switch",
 }
 SUPLA_FUNCTION_NONE = "NONE"
-SUPLA_CHANNELS = "supla_channels"
 SUPLA_SERVERS = "supla_servers"
+SUPLA_COORDINATORS = "supla_coordinators"
 
 SERVER_CONFIG = vol.Schema(
-    {vol.Required(CONF_SERVER): cv.string, vol.Required(CONF_ACCESS_TOKEN): cv.string}
+    {
+        vol.Required(CONF_SERVER): cv.string,
+        vol.Required(CONF_ACCESS_TOKEN): cv.string,
+        vol.Optional(CONF_SCAN_INTERVAL, default=10): cv.time_period,
```

Comment:
@mwegrzynek due to ADR10 we cant change this schema. just define a sane default and if/when this goes to config flow, it can be user customized then

Reply:
OK, fixed -- I've set the SCAN_INTERVAL to 10s